README.md

```diff
@@ -31,6 +31,8 @@ mailing list.
 
 Slack server: [kubeflow.slack.com](https://kubeflow.slack.com/)
 
+[Join Slack Channel](https://join.slack.com/t/kubeflow/shared_invite/enQtMjgyMzMxNDgyMTQ5LWUwMTIxNmZlZTk2NGU0MmFiNDE4YWJiMzFiOGNkZGZjZmRlNTExNmUwMmQ2NzMwYzk5YzQxOWQyODBlZGY2OTg)
```

Comment:
Where did you get this link from? Does the link expire?

Reply:
@jlewi I didn't remember actually, but this is the link I saved in my Google doc when I joined Kubeflow and this link works for me.

Reply:
@ewilderj Do you have any new invite links?

Reply:
@ScorpioCPH nope, I am not an admin on the Slack server.

Reply:
@amitkumarj441 Can you do a search to see what it takes make slack open so anyone can join? IIRC you actually have to use some third party app to auto-generate invites.

Reply:
@jlewi I can get a auto-generated invite link (I already have one for Rust India community longtime here: http://rustindia.herokuapp.com/) for anyone to join and yes, Thanks for reminding me about 3rd party apps :+1: :smile: 
Pinging you on slack to get this done now and will update here.